test/intl402/NumberFormat/constructor-roundingIncrement.js

```diff
@@ -39,7 +39,8 @@ for (const [value, expected] of values) {
     get roundingIncrement() {
       callOrder.push("roundingIncrement");
       return value;
-    }
+    },
+    minimumFractionDigits: 3
```

Comment:
Observation: This case, where maximumFractionDigits is not present and therefore defaults to 3, wasn't one that was considered when making the spec change. This is a little bit awkward, but I think it's fine, since in most cases roundingIncrement should have an explicit maximumFractionDigits value set (rather than relying on the implicit default).